api/v1/memo.go

```diff
@@ -313,6 +313,11 @@ func (s *APIV1Service) CreateMemo(c echo.Context) error {
 				createMemoRequest.Visibility = Private
 			}
 		}
+
+		// sanity check for visibility
+		if createMemoRequest.Visibility == Public && disablePublicMemos {
```

Comment:
Right now we allow admin to create public contents. And I think it's works good for me. 

So this check might be unnecessary.

https://github.com/usememos/memos/pull/2480/files#diff-e1e4065afaaae81d389a92c11ef9ac0fdbec4681bc20647843ce88dcf0193420L311

Reply:
Ah, missed that. Sounds good!